include/llvm/MC/MCStreamer.h

```diff
@@ -24,8 +24,6 @@
 #include "llvm/Support/SMLoc.h"
 #include <string>
 
-#include "llvm/Support/WinSAL.h"
```

Comment:
This one has _In_range(). I don't know if there are guides over whether required headers should be included in headers or if we rely on them being included in the right order.

Reply:
You can if you want. It seemed to build everywhere for me though in spite of that.